docs/intro/configuration.rst

```diff
@@ -1,4 +1,4 @@
-Configuration files
+Configuration
```

Comment:
`Configuration` sounds to broad. I think that `Configuration files` or `Models Configurations` could be better.

Reply:
resolved in https://github.com/deepmipt/DeepPavlov/pull/947/commits/919bd0e506404c6406ef0b2758edad0df0999f3c